oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/ExtractedTextCache.java

```diff
@@ -79,6 +80,7 @@
     private long totalTextSize;
     private long totalTime;
     private int preFetchedCount;
+    private StatisticsProvider statisticsProvider;
```

Comment:
For fields, I usually use "final" if possible. Here other fields are also not final, so I think it's fine.

Reply:
added final to Statistics provider, other references are also final in class above these primitive types